datalad/utils.py

```diff
@@ -369,6 +369,30 @@ def is_explicit_path(path):
     PurePosixPath,
 )
 
+if sys.version_info.major == 3 and sys.version_info.minor < 6:
+    # Path.resolve() doesn't have strict=False until 3.6
+    # monkey patch it -- all code imports this class from this
```

Comment:
>   all code imports this class from this module

I strongly doubt that's true, since I never did it ;-)
In any case, such an expectation needs to go into that design guidelines section in the docs we started.

Reply:
In that case, your code was adjusted to make it true nevertheless ;-)

Reply:
For that one point in time, yes. ;-)
Just saying: That assumption needs a clear statement. And probably should be picked up by extension developers as well.

Reply:
I am not much concerned with these scenarios. Import from this place and get stuff working on PY35, or deal with it manually. Any extension dev can decide for themselves, and also whether the want PY35 at all.